common/perception_utils/include/perception_utils/object_classification.hpp

```diff
@@ -17,6 +17,7 @@
 
 #include "autoware_auto_perception_msgs/msg/object_classification.hpp"
 
+#include <string>
```

Comment:
Just curious.

Where is this function with string argument used?
I'm not sure why the label is used as string in the implementation.

Reply:
Deep learning module sometime has string config like [CenterPoint](https://github.com/autowarefoundation/autoware.universe/blob/4be6931c811422f16d95c1460c046f2540d1f628/perception/lidar_centerpoint/config/centerpoint.param.yaml#L3), so this case need for converter.

Reply:
I don't know if having class names as a string is good or not, but I got it. Thank you.